pkg/etcdcli/interfaces.go

```diff
@@ -41,3 +41,9 @@ type UnhealthyMemberLister interface {
 type MemberStatusChecker interface {
 	MemberStatus(member *etcdserverpb.Member) string
 }
+
+// EtcdEndpointsGetter hides the implementation for getting etcd endpoints
+// so that custom fake endpoints can be used in tests
+type EtcdEndpointsGetter interface {
+	GetEtcdEndpoints() ([]string, error)
```

Comment:
rule of thumb is to use just Get() if the interface name is `Getter` and especially `EtcdEndpointGetter` :-)

Reply:
makes sense, updated. thanks for the nit, +1